content/en/docs/reference/command-line-tools-reference/feature-gates/readonly-apidata-volumes.md

```diff
@@ -0,0 +1,21 @@
+---
+# Removed from Kubernetes
+title: ReadOnlyAPIDataVolumes
+content_type: feature_gate
+
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta 
+    defaultValue: true
+    fromVersion: "1.18"
+    toVersion: "1.19"
+  - stage: deprecated
+    fromVersion: "1.10"
+    toVersion: "1.10"
+
+removed: true  
+---
+Set Secrets, ConfigMap, DownwardAPI and projected volumes to be mounted in read-only mode.
```

Comment:
```suggestion
Set [`configMap`](/docs/concepts/storage/volumes/#configmap),
[`secret`](/docs/concepts/storage/volumes/#secret),
[`downwardAPI`](/docs/concepts/storage/volumes/#downwardapi) and
[`projected`](/docs/concepts/storage/volumes/#projected)
{{< glossary_tooltip term_id="volume" text="volumes" >}} to be mounted read-only.

Since Kubernetes v1.10, these volume types are always read-only and you cannot opt out.
```

Not sure about that final line in the suggestion; if we don't like it, let's not include it.